Complex/complex.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <cmath>
+
+#include "Complex.h"
+
+using namespace std;
+
+Complex::Complex(double real, double imaginary) {
+    this->real = real;
+    this->imaginary = imaginary;
+}
+
+Complex::~Complex() {}
+
+double Complex::getReal() const {
+    return real;
+}
+
+double Complex::getImaginary() const {
+    return imaginary;
+}
+
+bool Complex::operator==(const Complex& other) const {
+    return real == other.real && imaginary == other.imaginary;
+}
+
+bool Complex::operator!=(const Complex& other) const {
+    return !(*this == other);
+}
+
+void Complex::operator+=(const Complex& other) {
+    real += other.real;
+    imaginary += other.real;
+}
+
+void Complex::operator-=(const Complex& other) {
+    real -= other.real;
+    imaginary -= other.real;
+}
+
+Complex Complex::operator+(const Complex& other) const {
+    Complex result = *this;
```

Comment:
Here and below: just create new Complex object  and set values regarding to `this` object and ` other` object